config/vufind/MyResearchMenu.ini

```diff
@@ -0,0 +1,71 @@
+; Optional configuration for user account menu. If there are no items, or this file is even not used. VuFind uses
+; default settings - which is same as you can see below.
+; The format of this configuration is as follows:
+; config key is a name of an item
+; array keys for every menu item could be:
```

Comment:
Might be worth noting required fields and default values here -- e.g. I assume that label and route are required, and status defaults to false.

Reply:
Yes, that's how it should work. I'll update the documentation.